### PR TITLE
LUS COVID demo

### PR DESCRIPTION
**Pull request by JulienVig**
Bug fixes to set up the LUS COVID demo for next week

- [x] Add test cases lus_covid (image_loader, validator, end to end training)
- [x] Fix node image_loader default channel error (default to 1 while it should be 3)
- [x] Add lus_covid task to CLI
- [x] Fine-tune hyper parameters
- [ ] Image dataset shuffling is done by hand while tfjs already implements it
- [x] Fix label encoding mistake (using tf.oneHot on 'positive' and 'negative' yields the same label for both class)
- [x] "Train Collaboratively" raises an error
- [x] Training loss and other metrics stay constant after the first few epochs
- [x] Model always predicts positive
- [x] ~~Training collaboratively with two web-client results in timeouts (server drop client contributions)~~
    - Addressed by #661 
- [x] ~~UI number of participants doesn't get updated as they join a task~~
    - Addressed by #661 
- [x] web-client training performance stays constant while training via node.js works well

Model used to only predict the majority class (~0.5 accuracy), by normalizing image values from 0-255 to 0-1 the training now yields >0.9 on both training and validation sets

### CODE REVIEW

File: server/src/tasks.ts [hunk 1]
```diff
@@ -31,40 +31,43 @@ export class TasksAndModels {
...
+    //   try {
+    //     await this.checkDigest(discoTask.digest, modelPath)
+    //   } catch (e) {
+    //     console.warn('removing model files at', modelPath)
+    //     await fs.rm(modelPath, { recursive: true, force: true })
+    //     throw e
+    //   }
+    // }
+    
+    // return model
```
Comment (tharvik) [inline server/src/tasks.ts]: avoid commented code

```suggestion
```

**Review by tharvik — APPROVED** [review]
génial, thanks for the hard work!

File: server/src/tasks.ts (hunk 1 above)
Comment (JulienVig) [reply server/src/tasks.ts]: Thanks! I think you reviewed an older commit? The code should be uncommented in the latest

Comment (tharvik) [reply server/src/tasks.ts]: ho right, that was an old commit; nvm then

**Comment by martinjaggi** [comment]
super nice, congrats!
